vpr/src/base/setup_vpr.cpp

```diff
@@ -1025,3 +1049,229 @@ static void do_reachability_analysis(t_physical_tile_type* physical_tile,
         }
     }
 }
+
+static void setup_vib_inf(const std::vector<t_physical_tile_type>& PhysicalTileTypes,
```

Comment:
I think this should be moved into a vib file in the tileable rr-graph if possible at all.